src/controllers/import.js

```diff
@@ -32,6 +32,20 @@ import { ImportJobDto } from '../dto/import-job.js';
  * @constructor
  */
 function ImportController(context) {
+  /**
+   * The import controller has a number of scopes that are required to access different parts of the
+   * import functionality. These scopes are used to validate the authenticated user has the required
+   * level of access.
+   * @type {{READ: 'imports.read', WRITE_ALL_DOMAINS: 'imports.write_all_domains',
+   * READ_ALL: 'imports.read_all', WRITE: 'imports.write'}}
+   */
+  const SCOPE = {
+    READ: 'imports.read', // allows users to read the import jobs created with their API key
+    WRITE: 'imports.write', // allows users to create new import jobs
+    READ_ALL: 'imports.read_all', // allows users to view all import jobs
+    WRITE_ALL_DOMAINS: 'imports.write_all_domains', // allows users to import across any domain
```

Comment:
Should we include this new scope in spacecat-shared as well? I'm planning to add it with my PR for https://jira.corp.adobe.com/browse/SITES-22756